### DISCUSSION TITLE
Weighting and scoring - Documentation questions

### DISCUSSION THREAD

**Discussion by geertjanknapen1** [statamic/cms#9571] [Q&A] [answered]
So the `Search options` part in the Search documentation (https://statamic.dev/search#search-options) states I can do some things regarding weighting and scoring;
> `match_weights`: An array of weights for each field to use when calculating relevance scores. Defaults to `null`.
> `score_threshold`: The minimum score required for a result to be included in the search results. Defaults to `null`.
> `property_weights`: An array of weights for each property to use when calculating relevance scores. Defaults to `null`.
> `sort_by_score`: Whether to sort the search results by relevance score. Defaults to `null`.

But, there are no concrete examples as to how to use this. 

It also states; (https://statamic.dev/search#local-driver)
> The local driver uses JSON to store key/value pairs, mapping fields to the content IDs they belong to. It lacks advanced features like weighting and relevance matching, but hey, It Just Works™. It's a great way to get a search started quickly.

So, my question is, are the search options available for the Local driver, or not? And if they are, how do I implement them?

**Comment by jasonvarga** [comment, marked as answer]
Thanks for pointing these out.

The local driver does support _some_ advanced features, and those settings have been moved to the right place on that page. I've also added an example of where to put them in your code.
